### ISSUE TITLE
Fatal error: A floating panel hidden must not be used by a user

### ISSUE THREAD

**Issue by Pivovar63** [SCENEE/FloatingPanel#100]
When I drag floating panel from half position up or down (it does not matter) and tap backdrop view by second finger it closes and immediately crashes with error:
`Fatal error: A floating panel hidden must not be used by a user`

It caused by handling pan gesture `cancelled` like `ended` here:
```
@objc func handle(panGesture: UIPanGestureRecognizer) {
...
case .ended, .cancelled, .failed:
                panningEnd(with: translation, velocity: velocity)
```

How can I fix it?

**Comment by SCENEE** [comment]
@Pivovar63 , Thanks for your report. It's almost the library's problem. I will investigate and try to fix it, but if you find a solution in library side, I welcome your PR 👍

**Comment by SCENEE** [comment]
I found a solution. This issue will be resolved by #102
